options/rtld/generic/linker.cpp

```diff
@@ -497,14 +519,17 @@ frg::expected<LinkerError, void> ObjectRepository::_fetchFromFile(SharedObject *
 #if MLIBC_MMAP_ALLOCATE_DSO
 	void *mappedAddr = nullptr;
 
-	if (mlibc::sys_vm_map(nullptr,
-			highest_address - object->baseAddress, PROT_NONE,
-			MAP_PRIVATE | MAP_ANONYMOUS, -1, 0, &mappedAddr)) {
-		mlibc::infoLogger() << "sys_vm_map failed when allocating address space for DSO \""
-				<< object->name << "\""
-				<< ", base " << (void *)object->baseAddress
-				<< ", requested " << (highest_address - object->baseAddress) << " bytes"
-				<< frg::endlog;
+	if(mlibc::sys_vm_map(nullptr,
+		   highest_address - object->baseAddress,
+		   PROT_NONE,
+		   MAP_PRIVATE | MAP_ANONYMOUS,
+		   -1,
+		   0,
+		   &mappedAddr)) {
```

Comment:
:x: This seems to be aligned with 4 spaces instead of tabs? We should not mix spaces and tabs, that is annoying to deal with.